cypress.json

```diff
@@ -3,7 +3,7 @@
   "ignoreTestFiles": "**/examples/*.spec.js",
   "trashAssetsBeforeRuns": false,
   "env": {
-    "target-id": "tprek:8100",
+    "resource-id": "tprek:8215",
```

Comment:
Changed this because when I tested the new api did not yet have the resource 'tprek:8100'